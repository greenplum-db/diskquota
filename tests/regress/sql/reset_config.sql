--start_ignore
\! gpconfig -c diskquota.naptime -v 10
\! gpstop -u
--end_ignore

SHOW diskquota.naptime;
