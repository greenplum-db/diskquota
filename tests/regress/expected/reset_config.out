SHOW diskquota.naptime;
 diskquota.naptime 
-------------------
 10
(1 row)

